pages/pipelines/artifacts.md

```diff
@@ -172,3 +176,7 @@ For an example, read [download an artifact from a specific step](#download-artif
 
 Alternatively, download the most recent matching file by using a glob pattern.
 For an example, read [download many artifacts](#download-artifacts-with-the-buildkite-agent-example-download-many-artifacts).
+
+### Artifacts are missing from retried jobs
+
+Artifacts from retried jobs are discarded by default, so the `buildkite-agent artifact download` command won't find them. To include artifacts from retried jobs in your search results, use `--include-retried-jobs` in the command.
```

Comment:
```suggestion
Artifacts from retried jobs are excluded by default, so the `buildkite-agent artifact download` command won't find them. To include artifacts from retried jobs in your search results, use `--include-retried-jobs` in the command.
```

Perhaps “discarded” could be read as “we delete artifacts when a job is retried”, and “excluded” is a nice inverse of the “include” in the CLI flag.